db/orders.js

```diff
@@ -0,0 +1,128 @@
+const {client} = require('./client');
+
+async function createOrder({
+    status, 
+    userId,
+    datePlaced,
+    }) {
+        const {rows: [order]} = await client.query(`
+            INSERT INTO orders(status, "userId", "datePlaced")
+            VALUES ($1, $2)
+            RETURNING *;
+        `, [status, userId, datePlaced])
+
+        return order;
+}
+
+async function getOrderById(id) {
+    try {
+        const{rows: [order]} = await client.query(`
+        SELECT *
+        FROM orders
+        WHERE orders.id = $1;
```

Comment:
should just be WHERE id = $1;

not referencing any other tables here, so no ambiguity with "id"